test/mxarray.jl

```diff
@@ -124,10 +124,10 @@ a_mx = mxarray(3.25)
 @test jscalar(a_mx) == 3.25
 delete(a_mx)
 
-a_mx = mxarray(@compat Int32(12))
-@test eltype(a_mx) == Int32
+a_mx = mxarray(12)
+@test eltype(a_mx) == Int
 @test size(a_mx) == (1, 1)
-@test jscalar(a_mx) == @compat Int32(12)
+@test jscalar(a_mx) == Int(12)
```

Comment:
Not sure what this was trying to test to begin with, but why change it from Int32 to Int?